alpha/lib/model/StorageProfile.php

```diff
@@ -363,8 +363,8 @@ public function isFlavorAssetConfiguredForExport(asset $flavorAsset)
 	public function isValidFileSync(FileSyncKey $key)
 	{
 		KalturaLog::log(__METHOD__ . " - key [$key], externalStorage id[" . $this->getId() . "]");
-		
-		list($kalturaFileSync, $local) = kFileSyncUtils::getReadyFileSyncForKey($key, false, false);
+
+		list($kalturaFileSync, $local) = kFileSyncUtils::getReadyFileSyncForKey($key, true, false);
```

Comment:
Avoid getting from remote storage if storageDc is locally

        $fetchFromRemote = true;
        if(in_array($this->getId(),kDataCenterMgr::getDcIds()))
        {
            $fetchFromRemote = false;
        }

		list($kalturaFileSync, $local) = kFileSyncUtils::getReadyFileSyncForKey($key, $fetchFromRemote, false);